sdk/servicebus/azure-messaging-servicebus/README.md

```diff
@@ -411,5 +415,9 @@ Guidelines](./../../../CONTRIBUTING.md) for more information.
 [subscription_concept]: https://docs.microsoft.com/azure/service-bus-messaging/service-bus-queues-topics-subscriptions#topics-and-subscriptions
 [topic_concept]: https://docs.microsoft.com/azure/service-bus-messaging/service-bus-messaging-overview#topics
 [wiki_identity]: https://github.com/Azure/azure-sdk-for-java/wiki/Identity-and-Authentication
+[java_development_kit]: https://docs.microsoft.com/en-us/java/azure/jdk/?view=azure-java-stable
+[AmqpRetryOptons]: ../../core/azure-core-amqp/src/main/java/com/azure/core/amqp/AmqpRetryOptions.java
+[service_bus_create]: https://docs.microsoft.com/azure/service-bus-messaging/service-bus-create-namespace-portal
+[dead-letter-queue]: https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-dead-letter-queues
```

Comment:
remove en-us from these links and let it redirect to user's locale.
also, all of the links are alphabetised.